### ISSUE TITLE
sizes plugin incompatible with rollup code-splitting

### ISSUE THREAD

**Issue by MichaelRando** [tivac/rollup-plugin-sizes#7]
Like in 
http://www.syntaxsuccess.com/viewarticle/lazy-loading-with-rollup

```
./src/App.js, ./src/ModuleA.js, ./src/ModuleB.js → ./dev...
[!] TypeError: Path must be a string. Received [ './src/App.js', './src/ModuleA.js', './src/ModuleB.js' ]
TypeError: Path must be a string. Received [ './src/App.js', './src/ModuleA.js', './src/ModuleB.js' ]
    at assertPath (path.js:28:11)
    at Object.dirname (path.js:1364:5)
    at Object.options (/Users/mscholz/exprment/webclient/node_modules/rollup-plugin-sizes/index.js:53:26)
    at /Users/mscholz/exprment/webclient/node_modules/rollup/dist/rollup.js:19063:31
    at Array.reduce (<anonymous>)
    at new Graph (/Users/mscholz/exprment/webclient/node_modules/rollup/dist/rollup.js:19061:32)
    at Object.rollup (/Users/mscholz/exprment/webclient/node_modules/rollup/dist/rollup.js:19630:23)
    at build (/Users/mscholz/exprment/webclient/node_modules/rollup/bin/rollup:3334:19)
    at /Users/mscholz/exprment/webclient/node_modules/rollup/bin/rollup:3970:20
    at next (/Users/mscholz/exprment/webclient/node_modules/rollup/bin/rollup:2781:16)
```

**Comment by tivac** [comment]
Well crud. Thanks for the report!

**Comment by tivac** [comment]
Fixed in `rollup-plugin-sizes@1.0.0`
